src/networking/controllers/example-controller.ts

```diff
@@ -8,18 +11,16 @@ import { API_ROUTES } from "networking/api-routes";
   Learn more about our networking architecture on:
   https://blog.xmartlabs.com/2020/07/09/frontend-architecture-and-best-practices/
 */
-class ExampleController {
-  static async getExamples(): Promise<Example[]> {
-    const response = await ApiService.get<RawExample[]>(API_ROUTES.EXAMPLE);
-    return (response || []).map<Example>(ExampleSerializer.deSerialize);
-  }
 
-  static createExample(example: Example) {
-    const serializedExample = ExampleSerializer.serialize(example);
-    return ApiService.post<RawExample>(API_ROUTES.EXAMPLE, {
-      body: JSON.stringify(serializedExample),
-    });
-  }
-}
+export const getExamples = async (): Promise<Example[]> => {
+  const response = await ApiService.get<RawExample[]>(API_ROUTES.EXAMPLE);
+  return response.map<Example>(deSerialize);
```

Comment:
Do we want to update these examples in the README? This one in particular in the **Networking** section

Reply:
_VERY_ good catch. Thanks! I'll work on documentation changes.